scripts/dappnode_access_credentials.sh

```diff
@@ -26,9 +27,16 @@ DAPPNODE_WELCOME_URL="http://welcome.dappnode"
 #1.FUNCTIONS#
 #############
 
-function dappnode_startup_delay () {
-  echo "Wait until DAppNode initializes..."
-  sleep 6
+function dappnode_startup_check () {
+  echo "Wait until DAppNode initializes (press ctrl+c to stop)..."
+  n=0
+  until [ "$n" -ge 8 ]
+  do
+    [ "$(docker inspect -f '{{.State.Running}}' ${DAPPMANAGER_CONTAINER} 2> /dev/null)" = "true" ] && break
+    n=$((n+1))
+    echo "$n try"
```

Comment:
This will look a very ugly:
```
1 try
2 try
3 try
4 try
5 try
6 try
```
Something we've done before that looks ok is to output a single dot so that the terminal will look like this after a few attempts

```
Wait until DAppNode initializes (press ctrl+c to stop).........
```

Reply:
Also, does this while loop has a maximum timeout? It should not wait forever

Reply:
done on last commit

Reply:
timeout is 8x8=64 secs. @tropicar Woult that time be enough in the worst case to detect a method ?